src/include/kdb.h.in

```diff
@@ -216,43 +188,31 @@ static inline Key *keyDup (const Key *source, elektraCopyFlags flags)
  *
  **************************************/
 
-KeySet *ksNew (size_t alloc, ...) ELEKTRA_SENTINEL;
-KeySet *ksVNew (size_t alloc, va_list ap);
-
-KeySet *ksDup (const KeySet * source);
-int ksCopy (KeySet *dest, const KeySet *source);
-
-uint16_t ksIncRef (KeySet * ks);
-uint16_t ksDecRef (KeySet * ks);
-uint16_t ksGetRef (const KeySet * ks);
-
-int ksClear (KeySet *ks);
-int ksDel (KeySet *ks);
-
-ssize_t ksGetSize (const KeySet *ks);
+ElektraKeySet * elektraKeySetNew (size_t alloc, ...) ELEKTRA_SENTINEL;
+ElektraKeySet * elektraKeySetVNew (size_t alloc, va_list ap);
 
-ssize_t ksAppendKey (KeySet *ks, Key *toAppend);
+ElektraKeySet * elektraKeySetDup (const ElektraKeySet * source);
+int elektraKeySetCopy (ElektraKeySet * dest, const ElektraKeySet * source);
 
-ssize_t ksAppend (KeySet *ks, const KeySet *toAppend);
-KeySet *ksCut (KeySet *ks, const Key *cutpoint);
+uint16_t elektraKeySetIncRef (ElektraKeySet * ks);
+uint16_t elektraKeySetDecRef (ElektraKeySet * ks);
+uint16_t elektraKeySetGetRef (const ElektraKeySet * ks);
```

Comment:
```suggestion
```

Should be moved to `libelektra-operations` together with `elektraKeyGetRef`. See also #4245.